kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -51,7 +50,7 @@
 
 def create_context(task) -> Context:
     dag = DAG(dag_id="dag")
-    tzinfo = pendulum.timezone("Europe/Amsterdam")
+    tzinfo = pendulum.timezone("Europe/Amsterdam")  # type: ignore
```

Comment:
Hmm, why is this needed, is pendulum’s type annotation falty?

Reply:
yea... sigh... so... i was getting `module is not callable` error. happened twice  in a row so not flakey.  maybe it was confused by the other `timezone`. it doesn't make any sense. but it's just a test so just ... went the easy route.  and frankly i have no clue how i would fix it otherwise.

Reply:
Ah that makes sense, there _is_ a `pendulum.timezone` subpackage.

Some of our code uses `pendulum.tz.timezone` instead and that seems to work fine.

Reply:
that's so weird though... how does that even work, if there is a function and a package that have the same name?  is it becaues it's a package and not a module that it works? and they just say "tie goes to the function"?

Reply:
It’s because Pendulum imports `timezone` in `__init__.py`, and then later overrides the name to a function:

https://github.com/sdispater/pendulum/blob/cc2db5d573f8936a475148bda3ff92057723fc48/pendulum/__init__.py

It is not unlike how Mypy dislikes this:

```python
a = 1
a = "2"  # type: ignore

def f(x: str) -> None:
    print(x)

f(a)  # error: Argument 1 to "f" has incompatible type "int"; expected "str"
```

Pendulum does not really care about strong typing, so everything’s fine from its perspective, but from Mypy’s perspective `pendulum.timezone` is still declared as a module (because that’s what appears first), not a function.

Reply:
oh, yeah, makes sense.  bad pendulum.

Reply:
ok i updated to use `tz`